plugin/hotswap-agent-spring-plugin/src/test/java/org/hotswap/agent/plugin/spring/mvc/SpringMvcTest.java

```diff
@@ -55,4 +57,18 @@ public void changeMappingAndMethodBody() throws Exception {
                 .andExpect(content().string("Hello World Swapped"));
     }
 
+    @Test
+    public void changeRequestMappingAndMethodBody() throws Exception {
```

Comment:
Hi! Even though I've never commented in this project before, may I make a suggestion, even if superfluous?
You could also add a test covering just a class change, with the mapping staying the same, just to cover that aspect as well.
Cheers.